docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,293 @@
+
+   - For a project document:
+
+     .. code-block:: bash
+
+        git checkout -b tickets/LCR-<N>
+        git push -u
+
+   - For a DM document:
+
+     .. code-block:: bash
+
+        git checkout -b tickets/RFC-<N>
+        git push -u
+
+   Replace ``<N>`` with the LCR or RFC number.
+
+4. When the CCB responds, they may ask for changes.
+   For minor and straightforward changes, you may commit changes to the release branch.
+   For more complex changes, or when multiple people  are working in parallel to address requests, use a ticket branch-based workflow instead.
+   Create the ticket branch from the head of the release branch and merge back to the release branch.
+
+   When issues are addressed, notify the CCB:
+
+   - For a project document, :ref:`create a new DocuShare upload <ccd-docushare-upload>` and notify the CCB.
+
+   - For a DM document, create a comment on the RFC confirming the changes and link to the ``https://<handle>.lsst.io/v/RFC-<n>`` landing page for the release branch.
+     You don't need to create intermediate DocuShare versions for the DM CCB.
+
+   Repeat this step for each round of CCB feedback.
+
+5. When the CCB approves document you create a release:
+
+   1. Commit to the head of the release branch with the following changes:
+
+      - Update `document's change record <https://lsst-texmf.lsst.io/lsstdoc.html#document-preamble>`_.
+        The Project librarian or DM release manager, through the CCB, determines the document's semantic version.
+      - Remove the ``draft`` option from the document class.
+      - Set the ``\date`` command using a YYYY-MM-DD format.
+
+   2. :ref:`Create a new DocuShare upload <ccd-docushare-upload>`.
+
+   3. Once the Project librarian or DM documentalist has uploaded the document and made it the new preferred version, create a :ref:`semantic version tag <ccd-semantic-tag>` at the same commit as the DocuShare tag:
+
+      .. code-block:: bash
+
+         git tag -a v<major>.<minor>
+         git push --tags
+
+      In your command, replace ``<major>.<minor>`` with the semantic version.
+
+      Format the Git tag message as:
+
+      .. code-block:: text
+
+         v<major.minor>
+
+         https://docushare.lsst.org/docushare/dsweb/Get/Version-<...>
+
+      The URL should point to the DocuShare version (same as the DocuShare tag).
+
+   4. Backport the amendment commits made on the release branch back to the ``master`` branch:
+
+      1. Create a user branch from the ``master`` branch:
+
+         .. code-block:: bash
+
+            git checkout master
+            git checkout -b u/<username>/v<major>.<minor>-backport
+
+      2. Cherry-pick commits from the release branch onto the new backport branch.
+         For example:
+
+         .. code-block:: bash
+
+            git cherry-pick <commit-sha>
+         
+         Also ensure that the document's change record is updated, but the ``draft`` option must remain in the document class command.
```

Comment:
it might be easier to achieve this if the draft removal and date setting is on a different commit to the change record modification.